src/js/script-loader.js

```diff
@@ -89,23 +89,31 @@ var LoaderProtoMethods = {
         }
 
         if (config.anonymous) {
+            var moduleName;
+
+            if (document.currentScript) {
```

Comment:
One thing to note is that this will always be falsy when running the tests, but the behaviour stays consistent for our tests in that case.

If we want to add a test manually loading anonymous modules and see that everything works as expected, we'll need to find a way to mock that somehow 😕